servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -495,6 +495,7 @@ HttpExecutionStrategyInfluencer buildStrategyInfluencerForClient(HttpExecutionSt
         return influencerChainBuilder.buildForClient(strategy);
     }
 
+    @Nullable
```

Comment:
Why the `@Nullable` required now?

Reply:
good call, not needed. was returning `null` at one point during development.